docs/topics/api/reviewers.rst

```diff
@@ -158,12 +158,24 @@ This endpoint allows you to compare two Add-on versions with each other.
 
 .. http:get:: /api/v4/reviewers/addon/(int:addon_id)/versions/(int:version_id)/compare_to/(int:version_id)/
 
-    Inherits most properties from :ref:`browse detail <reviewers-versions-browse-detail>` except ``file.content`` which is being
-    replaced by ``file.diff``.
+    Inherits most properties from :ref:`browse detail <reviewers-versions-browse-detail>`, except that most of the `file.entries[]` properties
+    can be `null` in case of a deleted file.
 
-    Given the complexity of the format, let's see the inline comments.
+    Properties specific to this endpoint:
 
-    The following output represents this git diff.
+    :>json array file.entries[]: The complete file-tree of the extracted XPI.
+    :>json string files.entries[].status: Status of this file, see https://git-scm.com/docs/git-status#_short_format
+    :>json int|null file.entries[].depth: Level of folder-tree depth, starting with 0.
+    :>json string file.entries[].filename: The filename of the file.
+    :>json string file.entries[].path: The absolute path (from the root of the XPI) of the file.
+    :>json string|null file.entries[].sha256: SHA256 hash.
+    :>json string|null file.entries[].mimetype: The determined mimetype of the file or ``application/octet-stream`` if none could be determined.
```

Comment:
Maybe this should be more clear that deletions are `null`? Otherwise it sounds like the mimetype would be `application/octet-stream`

Reply:
Fixed in 61da03f197